scripts/spheral-env.in

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env bash
 
-@CMAKE_INSTALL_PREFIX@/.venv/bin/python "$@"
+# Adding -B to prevent byte-compiling
+@CMAKE_INSTALL_PREFIX@/.venv/bin/python -B "$@"
```

Comment:
Could this trip up users that might import outside python modules that aren't previously byte compiled

Reply:
That shouldn't be a problem.   Throwing -B just prevents writing a .pyc to optimize the next import.  It shouldn't prevent importing files regardless of whether they have been previously byte-compiled or not.